docs/concepts/storage/persistent-volumes.md

```diff
@@ -473,7 +473,7 @@ spec:
 
 ## Raw Block Volume Support
 
-Static provisioning support for Raw Block Volumes is included as an alpha feature for v1.9. With this change are some new API fields that need to be used to facilitate this functionality. Currently, Fibre Channel is the only supported plugin for this feature.
+Static provisioning support for Raw Block Volumes is included as an alpha feature for v1.9. With this change are some new API fields that need to be used to facilitate this functionality. Currently, only Fibre Channel and Local Volume plugins support this feature.
```

Comment:
We'd like to avoid "currently" in the constantly changing flow of when these docs are read - can we update this to be specific to the version we're referencing? So perhaps instead of:
> Currently, only Fibre Channel and Local Volume plugins support this feature.

an alternative of:

> Kubernetes v1.9 supports only Fibre Channel and Local Volume plugins.

Reply:
@heckj Fixed.